Makefile

```diff
@@ -196,3 +196,15 @@ abi-gen:
 	solc --abi --pretty-json --overwrite -o distribution distribution/IDistribution.sol && \
 	solc --abi --pretty-json --overwrite -o staking staking/IStaking.sol && \
 	solc --abi --pretty-json --overwrite -o wasm wasm/IWasm.sol
+
+###############################################################################
+###                                Docker                                   ###
+###############################################################################
+get-heighliner:
+	go install github.com/strangelove-ventures/heighliner@latest
```

Comment:
Currently, heighliner@latest could cause build failing on some environments, heighliner@v1.7.2 could avoid the issue.